src/pages/api/loadUserActivity.ts

```diff
@@ -25,15 +27,20 @@ export default apiController<LoadUserActivityParams, LoadUserActivityResults>(as
 
   if (qStems) {
     qStemsNumber = qStems.length
+    qStems.forEach(qStem => {
+      generatedQStems.push(qStem._id.toString())
+    })
   }
+
   const qOptions = await OptionModel.find({
     class: { $eq: cid },
     learningObjective: { $regex: topic, $options: 'i' },
     author: user.id,
   })
 
   if (qOptions) {
-    qOptionsNumber = qOptions.length
+    const filteredOptions = qOptions.filter(option => !generatedQStems.includes(option.qstem.toString()))
```

Comment:
what does this do?

Reply:
> what does this do?

I tried preventing option count from counting options that a user creates to their own question (the option count only counts options that a user creates for another user's question) to create more collaborative environment :)

Reply:
I personally think (at least in the first deployment) we should control the point system through instructions. Let's ask Haesoo about this.